setup.py

```diff
@@ -4,7 +4,7 @@
     name='LalkaChat',
     version='0.0.1',
     packages=['', 'modules', 'modules.helpers', 'modules.messaging'],
-    requires=['requests', 'cherrypy', 'ws4py', 'irc', 'wxpython', 'cefpython'],
+    requires=['requests', 'cherrypy', 'ws4py', 'irc', 'wxpython', 'cefpython', 'sleekxmpp', 'pyasn1', 'pyasn1_modules', 'dnspython'],
```

Comment:
It will install **necessary** modules. For example, encryption is not necessary, xmpp could work either encrypted or unencrypted. You sure you want the second way? I hope not, so *pyasn1* is recommended. About *dnspython* - it's also not necessary, but without it *sleekxmpp* spam messages about dns problems and suggests *dnspython* as a solution.